src/app/modules/entry-live/components/live-filters/live-filters.component.ts

```diff
@@ -0,0 +1,71 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {FilterConfig} from "shared/components/filter/filter-base.service";
+import {FilterComponent} from "shared/components/filter/filter.component";
+import {ReportService} from "shared/services";
+import {LocationsFilterService} from "shared/components/filter/location-filter/locations-filter.service";
+import {DomainsFilterService} from "shared/components/filter/domains-filter/domains-filter.service";
+import {KalturaLogger} from "@kaltura-ng/kaltura-logger";
+import {animate, group, state, style, transition, trigger} from "@angular/animations";
+import {DateChangeEvent} from 'shared/components/date-filter/date-filter.service';
+import * as moment from 'moment';
+
+@Component({
+  selector: 'app-live-filters',
+  templateUrl: './live-filters.component.html',
+  styleUrls: ['./live-filters.component.scss'],
+  providers: [ReportService, LocationsFilterService, DomainsFilterService, KalturaLogger.createLogger('EntryFilterComponent')],
+  animations: [
+    trigger('state', [
+      state('visible', style({height: '*', opacity: 1})),
+      state('hidden', style({height: '0', opacity: 0})),
+      transition('* => visible', [
+        style({height: '0', opacity: 0}),
+        group([
+          animate(300, style({height: '*'})),
+          animate('400ms ease-in-out', style({'opacity': '1'}))
+        ])
+      ]),
+      transition('visible => hidden', [
+        style({height: '*', opacity: 1}),
+        group([
+          animate(300, style({height: 0})),
+          animate('200ms ease-in-out', style({'opacity': '0'}))
+        ])
+      ])
+    ])
+  ]
+})
+export class LiveFiltersComponent extends FilterComponent implements OnInit {
+  public filterConfig: FilterConfig = {};
+
+  @Input() set entryId(id: string) {
+    this.filterConfig.items = [{property: "entryIdIn", value: id}];
+  }
+
+  public predefinedBrowsers = [
```

Comment:
@viktor-kyryltsev-kaltura the browser and OS values should come from the report itself

Reply:
@amirch1 we don't have these apis for browser and OS and it's agreed with Nadav